pkg/network/prober/prober.go

```diff
@@ -174,6 +202,10 @@ func (m *Manager) doAsync(ctx context.Context, target string, arg interface{}, p
 
 		err = wait.PollImmediate(period, timeout, func() (bool, error) {
 			result, err = Do(ctx, m.transport, target, ops...)
```

Comment:
Isn't the `result` now somewhat redundant? Would just using the error return suffice now?